dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cs

```diff
@@ -17,6 +20,14 @@ internal static void AccessWindowsAppSDK()
             }
             catch (global::System.Exception e)
             {
+                if (global::System.Diagnostics.Debugger.IsAttached)
+                {
+                    global::System.Diagnostics.Debugger.Break();
+                }
+                else
+                {
+                    MessageBox(global::System.IntPtr.Zero, "You need to run the Windows App Runtime Installer and its MSIX components", "MddBootstrap", (uint)0x00000010L);
+                }
                 global::System.Environment.FailFast(e.Message);
             }
```

Comment:
We could implement the C++ goo in the bootstrap DLL and then none of this C# is needed.

I'd started working on a MddBootstrapInitialize2(). Let me steal parts of this PR over into it and share